src/axom/primal/geometry/NURBSPatch.hpp

```diff
@@ -358,7 +359,7 @@ class NURBSPatch
     , m_weights(weights)
   {
     auto pts_shape = pts.shape();
-    auto weights_shape = weights.shape();
+    // auto weights_shape = weights.shape();
```

Comment:
delete?

Reply:
@jcs15c -- `weights_shape` was declared but not used. Is it here for a reason? or mostly for symmetry w/ `pts_shape`? Should we remove it?

Reply:
This is apparently an oversight of mine from the `NURBSPatch` PR, it's supposed to check that the weights array matches the shape of the control point array. Sorry!

```suggestion
    auto weights_shape = weights.shape();
    
    SLIC_ASSERT(pts_shape == weights_shape);
```